addons/mail/static/src/js/systray/systray_activity_menu.js

```diff
@@ -150,12 +140,12 @@ const ActivityMenu = Widget.extend({
         if (data.domain) {
             domain = domain.concat(data.domain);
         }
-
+        const activityGroup = this.activityMenuView.messaging.models['ActivityGroup'].get(event.target.closest('.o_systray_activity').dataset['activityGroupLocalId']);
         this.do_action({
             type: 'ir.actions.act_window',
             name: data.model_name,
             res_model: data.res_model,
-            views: this._getViewsList(data.res_model),
+            views: activityGroup.irModel.availableWebViews.map(viewName => [false, viewName]),
```

Comment:
items were:
`kanban, list, form`
now are:
`kanban, list, form, activity`

Extra `activity` intentional?

Reply:
Items in activity menu necessarily have activities so the view is available for all of them. It doesn't matter much, it's just an extra choice of view for the user on the resulting action that was previously missing and is now there.